.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
+
+Include description and references to tickets or issues that this PR is related to
+
+## Checklist prior to adding reviewers
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I have run this successfully locally using `./build.ps1 compose`
+- [ ] I have added tests for my code and/or updated existing tests where my changes impacts them, or there is a valid explanation for not including tests in this PR
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] I created this branch from main and have integrated the latest changes
+- [ ] I conformed to the [Angular Commit Message Format](https://gist.github.com/brianclements/841ea7bffdb01346392c#commit-message-format) for all commit messages in this PR
+- [ ] I have ensured that the changes are backwards compatible, or I have ensured that `BREAKING CHANGE: <comment>` is included in the body of a commit message where backwards compatibility is broken
+- [ ] I have ensured that no package dependencies are targeting pre-release versions
+
+## Further comments
+
+Chance to add any other supporting information for the PR that is useful for the reviewer
+
+## Post review notes
+
+The PR owner is responsible for merging the PR and as a result, is responsible for ensuring the merge strategy used is performed correctly.
+
+Both rebasing and squashing are permitted into main for this repository. See below for supporting guidelines for doing this:
+
+### Rebasing
+
+- [ ] This PR is in the goldilocks zone. Not too big, not too small and all commits deserve to be on the main branch history
+- [ ] Breaking changes are suitably highlighted in the commit message
+
+### Squashing
```

Comment:
I'm generally against squashing commits, unless there is a good reason to do so...

Reply:
That is fine. In an ideal world changes would be small and could be rebased but squashes are a useful option for larger refactoring effort over time.

Is it possible for you to suggest an alternative wording that would suit you to save back and forth?